doc/README.md

```diff
@@ -0,0 +1,63 @@
+# SciPy Documentation
+
+## How to build the docs
+
+To build the html docs for local development (easiest), make sure you have built or installed
+SciPy itself first (e.g. with an in-place build via `setup.py develop` - for more details see the
+[Contributor Guide](http://scipy.github.io/devdocs/dev/contributor/contributor_toc.html#development-environment)).
+
+The first time you do this, also ensure to initialize and update submodules:
+```
+git submodule init
+git submodule update
+```
+
+Then build the html docs with:
+```
+make html
```

Comment:
1) `make html` or ` make html-scipyorg`? 

2) this will pick a wrong scipy unless scipy is installed with `python setup.py install` or a pip equivalent, no? Maybe do recommend `python runtests.py --shell` and then ` cd doc ; make html`

Reply:
I always prefer `make html`, simpler and 100% equivalent for working on anything but styling AFAIK.

> this will pick a wrong scipy unless scipy is installed with

That's no longer possible due to a check in the Makefile. And it's explained right above.

Reply:
> Maybe do recommend `python runtests.py --shell` and then ` cd doc ; make html`

Not a fan too be honest, something like `--shell` is just weird. If we want to make `python runtests.py --doc` work we can mention that, but in the absence of that I'd rather show how things actually work (1. build scipy, 2. build docs)

Reply:
It used to be that just building scipy was not enough, `cd doc && make html` would either not find scipy (and fail with a weird error) or pick up something installed.  It's been a while since I last did this, so what I say might be outdated. If things have been improved, great.

At any rate, this PR is an improvement over what was there anyway, and we can iterate if needed, so +1 from me.

Reply:
>If we want to make `python runtests.py --doc` work we can mention that,

It does work, see gh-10724.

Reply:
Great, thanks Peter - I'll add that.